<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager2</class>
 <widget class="QDialog" name="Manager2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="employeeId">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>640</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100000</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="eId">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #d7eba9;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1000</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>530</y>
     <width>761</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>610</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Ставка и сумма&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #ffafa5; border-radius: 15px; border: 0.2px solid #d7eba9; color: #ffffff; font-size: 9pt; font-weight: bold;}																
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="creditRequests_10">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>690</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Назначить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F9F9F9;
border-radius: 20px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="requestId">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100000</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Номер заявки&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="investRequest_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>Активные заявки
на вывод инвестиций</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getRequest_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>Взять заявку</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="tarifId">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>640</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100000</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="creditRequests_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>690</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100000</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #ebc834; border-radius: 5px; border: 0.2px solid #d7eba9; color: #ffffff; font-size: 9pt; font-weight: bold;}																
QPushButton{background: #d9d9d9; border-radius: 5px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>501</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F9F9F9;
border-radius: 20px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="investId">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100000</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F9F9F9;
border-radius: 20px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="clientId">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100000</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Номер клиента&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clientInfoButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>Информация о клиенте</string>
    </property>
   </widget>
   <widget class="QPushButton" name="creditRequests_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 8pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 8pt; font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>История инвестиций</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="extraSum">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100000</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>610</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Номер тарифа&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="creditRequests_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Тарифы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="creditRequests_9">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>690</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F9F9F9;
border-radius: 20px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="compute">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 5px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 5px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>Счет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="resSum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100000</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="creditRequests_11">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>570</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 9pt; font-weight: bold;}													
QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Активность работников</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>640</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100000</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>610</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;ID и премия&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Номер инвестиции&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>560</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F9F9F9;
border-radius: 20px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="checkData">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #b9de6f; border-radius: 5px; border: 0.2px solid #d7eba9; color: #ffffff; font-size: 9pt; font-weight: bold;}	
																	QPushButton{background: #d9d9d9; border-radius: 5px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Доходы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="period">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #d9ebff;
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100000</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>label_6</zorder>
  <zorder>employeeId</zorder>
  <zorder>eId</zorder>
  <zorder>line</zorder>
  <zorder>label_12</zorder>
  <zorder>cancel</zorder>
  <zorder>creditRequests_10</zorder>
  <zorder>groupBox</zorder>
  <zorder>tarifId</zorder>
  <zorder>creditRequests_6</zorder>
  <zorder>sum</zorder>
  <zorder>cancel_3</zorder>
  <zorder>investId</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>extraSum</zorder>
  <zorder>label_11</zorder>
  <zorder>tableView</zorder>
  <zorder>creditRequests_8</zorder>
  <zorder>creditRequests_9</zorder>
  <zorder>comboBox</zorder>
  <zorder>creditRequests_11</zorder>
  <zorder>rate</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>checkData</zorder>
  <zorder>period</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
